wpilibc/src/main/native/include/frc/simulation/SolenoidSim.h

```diff
@@ -18,8 +18,34 @@ class SolenoidSim {
   SolenoidSim(PneumaticsModuleType type, int channel);
   ~SolenoidSim() = default;
 
-  bool GetOutput() const;
-  void SetOutput(bool output);
+  bool Get() const;
+  void Set(bool output);
```

Comment:
I'm assuming these are renamed for consistency with the hardware driver; The original functions will need to be marked for deprecation. Documentation should be added as well.